net.wz2100.wz2100.yaml

```diff
@@ -44,16 +58,33 @@ modules:
     sources:
       - type: git
         url: https://github.com/fribidi/fribidi
-        tag: v1.0.8
-        commit: 2e9bd91cfd857c5b78a210d8f24862a4bb704059
+        tag: v1.0.9
+        commit: f9e8e71a6fbf4a4619481284c9f484d10e559995
       - type: patch
         path: disable-docs.patch
-      
+
+  - name: libsodium
+    sources:
+      - type: archive
+        url: https://download.libsodium.org/libsodium/releases/libsodium-1.0.18.tar.gz
+        sha256: 6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1
+
+  - name: miniupnpc
```

Comment:
It should not be necessary to include this section downloading miniupnpc. WZ's CMake build system will handle this automatically, with an in-tree copy (that is substantially newer than 2.1).

Reply:
Thank you - I removed it; build is running